Common/API/LiveAlgorithmResults.cs

```diff
@@ -0,0 +1,208 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using QuantConnect.Api;
+using QuantConnect.Orders;
+
+namespace QuantConnect.API
+{
+    /// <summary>
+    /// Details a live algorithm from the "live/read" Api endpoint
+    /// </summary>
+    public class LiveAlgorithmResults : RestResponse
+    {
+        /// <summary>
+        /// Represents data about the live running algorithm returned from the server
+        /// </summary>
+        public LiveResults LiveResults { get; set; }
+    }
+
+    /// <summary>
+    /// Holds information about the state and operation of the live running algorithm
+    /// </summary>
+    public class LiveResults
```

Comment:
@jaredbroad - I did look for an existing class in the solution before creating this one to deserialize the results.  It's such a minor difference, but LiveResult uses a struct called ChartPoint (Common\Charting.cs) that is incompatible with the response from the Api.  

ChartPoint is what holds the x and y values.  For some reason, when you fetch the results from the Api, the first y value in the series of x and y values is null.  This minor difference throws an error when you try to deserilize.  I tried making the "y" property nullable, but this would mean fixing code everywhere that uses this struct.  

I didn't like it either, but the easiest path to getting the Api response deserialized was to add this class.  I'm all ears for a better way though.

![nooooooooo](https://cloud.githubusercontent.com/assets/2603162/18966832/c84a87c4-8650-11e6-93e0-2fdf5de8fe26.png)